ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/CommandlineParser.java

```diff
@@ -53,65 +54,54 @@ public boolean parseCommandLine(final String... args) {
     final CommandLine commandLine = new CommandLine(baseCommand);
     commandLine.setCaseInsensitiveEnumValuesAllowed(true);
     commandLine.registerConverter(Level.class, Level::valueOf);
+    commandLine.setOut(stdOut);
+    commandLine.setErr(stdErr);
+    commandLine.setExecutionExceptionHandler(this::handleExecutionException);
+    commandLine.setParameterExceptionHandler(this::handleParseException);
 
     for (final SignerSubCommand subcommand : signers) {
       commandLine.addSubcommand(subcommand.getCommandName(), subcommand);
     }
 
-    try {
-      commandLine.parseWithHandlers(new RunLast().useOut(output), new ExceptionHandler<>(), args);
-      return true;
-    } catch (final ParameterException ex) {
-      handleParameterException(ex);
-    } catch (final ExecutionException ex) {
-      commandLine.usage(output);
-    } catch (final TransactionSignerInitializationException | InitializationException ex) {
-      // perform no-op (user output already supplied in ExceptionHandler)
-    } catch (final Exception ex) {
-      LOG.error("Ethsigner has suffered an unrecoverable failure", ex);
-      output.println("Ethsigner has suffered an unrecoverable failure " + ex.toString());
-    }
-    return false;
+    final int resultCode = commandLine.execute(args);
```

Comment:
this feels like a bigger change than this PR needs to deal with, can we break the CMDline _functional_ changes out to a different PR?

Reply:
reverted